app/models/contact.rb

```diff
@@ -1,6 +1,10 @@
 class Contact < ActiveRecord::Base
+  EMBEDDED_CONTACT_REGEXP = /\[Contact\:([0-9]+)\]/
```

Comment:
It is being used in 2 places: `GovspeakHelper` which is the view helper, and `GovspeakHelper::ContactsExtractor` as well.